Documentation/coding-guidelines/breaking-change-rules.md

```diff
@@ -158,16 +158,17 @@ Breaking Change Rules
 * Adding `virtual` to a member  
 > Make note, that marking a member virtual might cause previous consumers to still call the member non-virtually.
 
-* Introducing or removing an override
-> Make note, that introducing an override might cause previous consumers to skip over the override when calling `base`.
-
 &#10007; **Disallowed**
 * Adding an member to an interface
 
 * Adding an abstract member to a type when there _are accessible_ (`public` or `protected`) constructors and the type is not `sealed`
 
 * Adding a constructor to a class which previously had no constructor, without also adding the default constructor
 
+* Introducing or removing an override to non-sealed `protected` or `protected internal` types
+> Introducing an override will cause previous consumers to skip over the override when calling `base` if not compiled against the new version.
+> Removing an override will cause previous consumers that call `base` to throw `TypeLoadException` if not compiled against the new version.
```

Comment:
This rule as written is oversimplification of the problem. You can remove the override and still be just fine in number of cases.

BTW: https://blogs.msdn.microsoft.com/ericlippert/2010/03/29/putting-a-base-in-the-middle/ has the detailed description of this issue.